src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+Title: Basic client usage - tutorial
+TODO:  Warning: Ubuntu release versions hardcoded
+       Include links to planned tutorials (tutorial-bundles.md, tutorial-constraints.md)
+
+# Basic client usage - tutorial
+
+The goal of this tutorial is to give new Juju operators a solid introduction to
+the command line client. In so doing, several important concepts are
+introduced. It also discusses the use of command prefixes and aliases. Further
+reading suggestions are included at the end.
+
+## Prerequisites
+
+The following prerequisites are assumed as a starting point for this tutorial:
+
+ - You're using Ubuntu 18.04 LTS (Bionic).
+ - Juju (stable snap channel) is installed. See the [Installing Juju][install]
+   page.
+ - You have chosen a backing cloud and have created a controller for it. Refer
+   to the [Clouds][clouds] page to get started with a cloud and controller.
+ 
+This guide uses LXD (`v.3.9`) as a backing cloud due to its accessibility and
+low resource usage. Choose a local LXD cloud if you're not sure what to use
+(see [Using LXD with Juju][clouds-lxd]).
+
+## Command prefixes and aliases
+
+As you gain more experience with the client, you will encounter command
+prefixes whose meanings may not be immediately apparent.
+
+A good example of this are the `destroy-` and `remove-` prefixes, such as in
+the commands `destroy-model` and `remove-user`. These two prefixes differ in
+terms of severity. Generally speaking, `destroy-` indicates a destructive
+action that is difficult to reverse whereas `remove-` does not.
+
+There is an even more drastic-sounding "remove" prefix: `kill-`, but it is
+featured in a sole command: `kill-controller`. The latter command differs
+from `destroy-controller` in that it has the ability to terminate the
+controller machine directly via the cloud provider, without cleaning up any
+machines that may be running in workload models.
+
+The `show-` prefix is used to drill down into an object to reveal details. It
+is akin to the verb "describe".
+
+There are command aliases, that typically employ a prefix, that point to their
+corresponding commands. This practice is for simplifying the life of the
+operator. The `list-` alias/prefix is a good example, where `list-clouds`
+points to `clouds`.
+
+The list of aliases can be obtained in this way:
```

Comment:
I think for aliases I'd just note that list-xxx can typically be shortened to just the entity e.g. list-clouds to clouds. Then note there are a few other aliases and that you can find them in the help commands. I think this paragraph is odd because it again starts off with exceptions vs just stating "there are list commands for the entities in Juju, you can normally shorten the command by leaving off the list because there are aliases setup for them. By the way there are other aliases out there, here's how to find them"

Reply:
I'll reword.